app.js

```diff
@@ -44,25 +47,74 @@ app.Views.TodoAppView = Backbone.View.extend({
         this.todoContent.append(this.subviews.todoListView.render());
 
         return this.$el;
+    },
+    createTodoOnEnter: function(e) {
```

Comment:
nit: favour `event` instead of `e`. It just makes your code clearer and easier to read.